.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+sudo: required
+addons:
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+node_js:
+  - "7"
+before_install:
+  - npm setup
```

Comment:
is this command exist ? 

If was not too old right now, I think this command should be : `npm install`

Reply:
npm setup is a script function that already there in package.json.
Please see all change.
It should already there in package.json